src/pci_mav.cpp

```diff
@@ -239,7 +242,7 @@ PCIMAV::generateTrajectoryMarkerArray(
     mo.pose.orientation.w = ti_q.w;
     m_arr->markers.push_back(mo);
   }
-  m_arr->markers.empty() ?: ++mo.id;
+  if(!m_arr->markers.empty()) ++mo.id;
```

Comment:
Please write in the next line with curly braces

Reply:
Resolved with [this commit](https://github.com/ntnu-arl/pci_mav/pull/1/commits/f95de6b8c469735fe3711a6ae286113abeba147f)